src/crate/crash/command.py

```diff
@@ -360,6 +367,8 @@ def _try_exec_cmd(self, line):
                     message = cmd(self, *words[1:])
                 else:
                     message = cmd(*words[1:])
+            except ProgrammingError as e:
+                raise e
```

Comment:
print error rather than raising exception?

Reply:
this is necessary because otherwise the REPL (which calls the user-input prompt) won't receive any exception.

Reply:
got it

Reply:
Could add a comment along the lines of `# repl needs to handle authorization errors`